app/src/main/res/values-lt/strings.xml

```diff
@@ -92,13 +103,6 @@
     <!-- Text for the negative action button to dismiss the Close Tabs Banner. -->
     <string name="tab_tray_close_tabs_banner_negative_button_text">Paslėpti</string>
 
-    <!-- Text for the banner message to tell users about our grid view feature. -->
-    <string name="tab_tray_grid_view_banner_message">Pakeiskite atvertų kortelių išdėstymą. Eikite į nuostatas, ir pasirinkite tinklelį prie kortelių rodinio.</string>
-    <!-- Text for the positive action button to go to Settings for auto close tabs. -->
-    <string name="tab_tray_grid_view_banner_positive_button_text">Eiti į nuostatas</string>
-    <!-- Text for the negative action button to dismiss the Close Tabs Banner. -->
-    <string name="tab_tray_grid_view_banner_negative_button_text">Paslėpti</string>
```

Comment:
I see these Strings still used in code - https://github.com/mozilla-mobile/fenix/blob/releases_v90.0.0/app/src/main/java/org/mozilla/fenix/tabstray/TabsTrayInfoBannerBinding.kt#L69-L71
([permalink](https://github.com/mozilla-mobile/fenix/blob/releases_v90.0.0/app/src/main/java/org/mozilla/fenix/tabstray/TabsTrayInfoBannerBinding.kt#L69-L71))

Reply:
cc: @Delphine

Reply:
I don't know how to fix this, but @st3fan was talking about this on Slack with jcristau Tuesday.
@st3fan What's the right way to go about getting these in again? Just wait for the next beta to pick it up?